src/controllers/userController.js

```diff
@@ -100,4 +100,54 @@ const logOut = (req, res) => {
   }
 };
 
-module.exports = { signUp, logIn, logOut };
+/******************************************************
+ * @UPDATEUSER
+ * @route /api/v1/updateuser
```

Comment:
I'm really curious what is type of comment called? And where are you getting all these keywords (@route, @method, @description, @body, @returns) from?

Reply:
I searched and found that there is [JSDoc](https://jsdoc.app/). But I couldn't find `@route`, `@method`, `@description`, `@body`, `@returns` in the allowed methods. Could you clarify it?

Reply:
It's not method😅 😅
 Just a way to comment my code .